src/Domain/Entity/RelatedLink.php

```diff
@@ -65,6 +66,45 @@ public function getType(): string
 
     public function isExternal(): bool
     {
+        if ($this->isExternal === null) {
+            $this->setUriMetadata();
+        }
+
+        // Assert to make PHPStan confident that setUriMetadata sets isExternal
+        // to a not-null value. Without this PHPStan raises a wanring
+        // See https://github.com/phpstan/phpstan/issues/856
+        assert($this->isExternal !== null);
+
         return $this->isExternal;
     }
+
+    public function getHost(): string
```

Comment:
Why do we need this method?

Reply:
Because in ds2013 layouts it shows the hostname if the link is external (e.g. the related links on the promos at http://www.bbc.co.uk/programmes/p010772m)